lib/heex_formatter/phases/format.ex

```diff
@@ -212,4 +215,61 @@ defmodule HeexFormatter.Phases.Format do
         ~s(#{attr})
     end
   end
+
+  defp format_eex(code, opts) do
```

Comment:
```suggestion
  defp format_eex(code, indentation) do
```

Let's send the indentation or you prefer, the current state. Wdyt?

Reply:
lets send the current state, it might be helpful to get other info we may need, max line length and so on